connector/docs/sobjects.md

```diff
@@ -97,6 +183,167 @@ Given below is a sample request that can be handled by the describeSobjects oper
+                <activateable>false</activateable>
+                <childRelationships>
+                    <cascadeDelete>false</cascadeDelete>
+                    <childSObject>Account</childSObject>
+                    <deprecatedAndHidden>false</deprecatedAndHidden>
+                    <field>ParentId</field>
+                    <relationshipName>ChildAccounts</relationshipName>
+                </childRelationships>
+                .
+                .
+            </result>
+            <result>
+                <activateable>false</activateable>
+                <childRelationships>
+                    <cascadeDelete>false</cascadeDelete>
+                    <childSObject>AcceptedEventRelation</childSObject>
+                    <deprecatedAndHidden>false</deprecatedAndHidden>
+                    <field>RelationId</field>
+                    <relationshipName>AcceptedEventRelations</relationshipName>
+                </childRelationships>
+                .
+                .
+            </result>
+        </describeSObjectsResponse>
+    </soapenv:Body>
+</soapenv:Envelope>
+```
+###### Related Salesforce documentation
 
 [https://developer.salesforce.com/docs/atlas.en-us.api.meta/api/sforce_api_calls_describesobjects.htm](https://developer.salesforce.com/docs/atlas.en-us.api.meta/api/sforce_api_calls_describesobjects.htm)
+
+### Sample configuration
+
+Following example illustrates how to connect to Salesforce with the init operation and describeGlobal operation.
+
+1. Create a sample proxy as below :
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<proxy xmlns="http://ws.apache.org/ns/synapse"
+       name="salesforce_describeGlobal"
+       startOnLoad="true"
+       statistics="disable"
+       trace="disable"
+       transports="http,https">
+   <target>
+      <inSequence>
+         <property xmlns:ns="wso2.connector.salesforce"
+                   expression="//ns:loginUrl/text()"
+                   name="loginUrl"/>
+         <property xmlns:ns="wso2.connector.salesforce"
+                   expression="//ns:username/text()"
+                   name="username"/>
+         <property xmlns:ns="wso2.connector.salesforce"
+                   expression="//ns:password/text()"
+                   name="password"/>
+         <property xmlns:ns="wso2.connector.salesforce"
+                   expression="//ns:blocking/text()"
+                   name="blocking"/>
+         <salesforce.init>
+            <loginUrl>{$ctx:loginUrl}</loginUrl>
+            <username>{$ctx:username}</username>
+            <password>{$ctx:password}</password>
+            <blocking>{$ctx:blocking}</blocking>
+         </salesforce.init>
+         <salesforce.describeGlobal/>
+         <respond/>
+      </inSequence>
+      <outSequence>
+         <send/>
+      </outSequence>
+   </target>
+   <description/>
+</proxy>                              
+```
+2. Create a xml file called describeGlobal.xml containing the following xml:
```

Comment:
Create an XML file named describeGlobal.xml and copy the XML configurations given below: